### ISSUE TITLE
[Usable - ADF - Azure data factory] Elements have insufficient contrast ratio on Azure data factory window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3840]
**User Experience:**
Visual impaired user will find difficulty to see the control or element if that control or element have insufficient color contrast as user won't be able to perform functionality over such control.

**Note:** User credentials should NOT be included in the bug.

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it.

**Repro Steps:​**
Launch Microsoft Azure Storage Explorer app.
Navigate and select Add data factory control at left panel.
Navigate to visually appearing element 'Azure Data factory'.
Verify contrast ratio of 'Azure Data factory' element.

**Actual Result:**
Elements 'Azure Data factory' have just below 4.5:1 foreground and background colors on Azure data factory window and due to this, visual impaired user will might get difficulty to see the element/control.

**Expected Result:**
Elements 'Azure Data factory' must have sufficient 4.5:1 contrast between foreground and background colors on Azure data factory window so that visual impaired user could easily the control or element.

**Reference Links:**

- **External Bug Process:** If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://apc01.safelinks.protection.outlook.com/?url=https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906839611&sdata=D5fhtDtz6f9O9hGT6oPoowXq0L/TCfaejH73thazOvA%3D&reserved=0). **“HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.**
- **Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process related queries.**
![Usable_Azure data factory](https://user-images.githubusercontent.com/72871089/101134986-2eb76380-3631-11eb-9a0a-1403e024d2ff.png)
